bool override_status
bool joystick_status
